rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -11,8 +11,7 @@ info:
   version: ${project.version}
   title: Query API
   description: >
-    The Query API is defined to provide clients with the ability to
-    query for stored datasets.
+    The Query API is defined to provide clients with the ability to query on datasets.
```

Comment:
❓ Is this true?  This makes it sound like we return a list of datasets, but we really return a list of Results (metacardLocation and IrmLocation).

Reply:
Is says query on datasets. It does not say that it returns datasets.

Reply:
Maybe it's just me, but when I query on "something", I would expect to get a list of that "something" back. 🤷‍♂

Reply:
Do you have a suggestion for a better description?

Reply:
I'm just saying that if we are going to use datasets in our system, we should return datasets not parts of datasets.  It looks like a bad design to me, but maybe we are just stuck with it.

Reply:
Do you agree that updating the API to return datasets is outside the scope of this PR? If yes, do you think this description needs to be updated in this PR?

Reply:
I agree that updating the API to return datasets is outside the scope of this PR.  I guess we can leave the description as-is.